File: sandbox.controller/src/kz/greetgo/sandbox/controller/model/ClientInfo.java
package kz.greetgo.sandbox.controller.model;

public class ClientInfo {
    private String id;
    private String surname;
    private String name;
    private String patronymic;
    private String charm;
    private int age;
    private int totalBalance;
    private int minBalance;
    private int maxBalance;

    public ClientInfo(String id,
                    String surname,
                    String name,
                    String patronymic,
                    String charm,
                    int age,
                    int totalBalance,
                    int minBalance,
                    int maxBalance) {
        this.id = id;
        this.surname = surname;
        this.name = name;
        this.patronymic = patronymic;
        this.charm = charm;
        this.age = age;
        this.totalBalance = totalBalance;
        this.minBalance = minBalance;
        this.maxBalance = maxBalance;
    }

    public ClientInfo() {

    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getSurname() {
        return surname;
    }

    public void setSurname(String surname) {
        this.surname = surname;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPatronymic() {
        return patronymic;
    }

    public void setPatronymic(String patronymic) {
        this.patronymic = patronymic;
    }

    public String getCharm() {
        return charm;
    }

    public void setCharm(String charm) {
        this.charm = charm;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public int getTotalBalance() {
        return totalBalance;
    }

    public void setTotalBalance(int totalBalance) {
        this.totalBalance = totalBalance;
    }

    public int getMinBalance() {
        return minBalance;
    }

    public void setMinBalance(int minBalance) {
        this.minBalance = minBalance;
    }

    public int getMaxBalance() {
        return maxBalance;
    }

    public void setMaxBalance(int maxBalance) {
        this.maxBalance = maxBalance;
    }
}

File: sandbox.client/front/ts/model/ClientInfo.ts
import {Charm} from "./Charm";

export class ClientInfo {
  public id: string;
  public surname: string | null;
  public name: string | null;
  public patronymic: string | null;
  public charm: Charm | null;
  public age: number/*int*/;
  public totalBalance: number/*int*/;
  public minBalance: number/*int*/;
  public maxBalance: number/*int*/;

  public assign(o: any): ClientInfo {
    this.id = o.id;
    this.surname = o.surname;
    this.name = o.name;
    this.patronymic = o.patronymic;
    this.charm = o.charm;
    this.age = o.age;
    this.totalBalance = o.totalBalance;
    this.minBalance = o.minBalance;
    this.maxBalance = o.maxBalance;
    return this;
  }

  public static copy(a: any): ClientInfo {
    let ret = new ClientInfo();
    ret.assign(a);
    return ret;
  }
}
